src/app/pages/StakingPage/Features/ValidatorList/index.tsx

```diff
@@ -51,7 +52,17 @@ export const ValidatorList = memo((props: Props) => {
   const columns: IDataTableColumn<Validator>[] = [
     {
       name: '',
-      cell: datum => <img src={process.env.PUBLIC_URL + '/logo192.png'} height="16px" width="16px" alt="" />,
+      cell: datum => (
+        <img
+          src={
+            datum.media?.logotype && isWebUri(datum.media?.logotype)
```

Comment:
```suggestion
            datum.media?.logotype && isWebUri(datum.media.logotype)
```

could we do this? with the left side of the `&&` effectively guarding for `datum.media`

Reply:
Of course ;)